test/locationscale.jl

```diff
@@ -2,11 +2,11 @@ function test_location_scale(
     rng::Union{AbstractRNG, Missing},
     μ::Real, σ::Real, ρ::UnivariateDistribution, dref::UnivariateDistribution,
 )
-    d = LocationScale(μ,σ,ρ)
+    d = μ + σ * ρ
```

Comment:
This should be

```suggestion
    d = Distributions.AffineDistribution(μ, σ, ρ)
```

The main point is to use the constructor here and compare it with the constructions based on `+` and `*` below.